cluster-provision/k8s/scripts/provision.sh

```diff
@@ -38,14 +40,11 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
        https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
 yum install -y \
-  docker-common-1.13.1-75.git8633870.el7.centos.x86_64 \
-  origin-docker-excluder-3.11.0-1.el7.git.0.62803d0.noarch \
-  python-docker-py-1.10.6-4.el7.noarch \
-  docker-client-1.13.1-75.git8633870.el7.centos.x86_64 \
-  cockpit-docker-176-2.el7.centos.x86_64 \
-  docker-1.13.1-75.git8633870.el7.centos.x86_64 \
-  python-docker-pycreds-1.10.6-4.el7.noarch
+  docker-1.13.1 \
```

Comment:
how is this change related to the patch?

Reply:
I had to update the install as the packages requested previously weren't available any more. We agreed upon just installing whatever is available instead of using the exact packages.

Reply:
Yep, and the background: We were pinning them back then because of bugs.